ubo_app/display.py

```diff
@@ -41,51 +40,59 @@
         cs=cs_pin,
         dc=dc_pin,
         rst=reset_pin,
-        baudrate=60000000,
+        baudrate=70000000,
```

Comment:
@sassanh can we push this higher or is this the limit?

Reply:
@mehrdadfeller I'm not sure, I have a very vague memory of you telling me based on data-sheets the baud rate of this display is 60M, but since there were no reports of artifacts in the past year, I thought it should be safe to increase it by 10M as it has benefits for spinner's fps.
In my local tests it can be increased to 95M without any issues, but I think we should have a safety margin to make sure things work in noisy environments and with not-perfectly-assembled devices.